app/utils/wikiFetch.tsx

```diff
@@ -52,5 +52,14 @@ export async function getWikipediaContestantData(): Promise<IWikipediaContestant
         return aContestant
     }).toArray()
 
-    return contestantData
+    const result = filterEmptyContestants(contestantData)
+
+    return result
+}
+
+export function filterEmptyContestants(contestantList: IWikipediaContestantData[]): IWikipediaContestantData[] {
```

Comment:
This is only exported so that it can be tested. Certainly open to other suggestions here.

Reply:
Outside of running tests in this file, I believe you need to export it.

Reply:
Oh? How come? it is only used, right in this module.